src/Runner.Listener/MessageListener.cs

```diff
@@ -13,10 +13,7 @@
 using System.Runtime.InteropServices;
```

Comment:
All changes that are related to SPS/Token conversion is removed from this file.

Reply:
discussed offline... diff new code against pre-migration code === identical